dev-docs/bidders/themoneytizer.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: The Moneytizer
+description: Prebid The Moneytizer Bid Adapter
+biddercode: themoneytizer
+media_types: display
```

Comment:
The code specifies banner, video and native as supported. Shouldn't this here reflected here as well?

Reply:
Hi @muuki88
Thanks for the review, it was a mistake in the adapter code, it's fixed now